app/javascript/dashboard/components/ui/Dropdown/DropdownListItemButton.vue

```diff
@@ -8,6 +8,14 @@ defineProps({
     type: Boolean,
     default: false,
   },
+  leftIcon: {
```

Comment:
Do we need to call it `leftIcon`, we can just call it `icon`, same with `leftIconColor`